x-pack/plugins/fleet/server/services/audit_logging.ts

```diff
@@ -21,15 +20,14 @@ class AuditLoggingService {
    */
   public writeCustomAuditLog(args: AuditEvent) {
     const securitySetup = appContextService.getSecuritySetup();
-    let auditLogger: AuditLogger | undefined;
 
-    const request = getRequestStore().getStore();
+    // const request = getRequestStore().getStore();
```

Comment:
I think the `AsyncLocalStorage` should be a singleton like in the example [here](https://nodejs.org/api/async_context.html), and we are creating a new instance on each invocation of `getRequestStore()` from `writeCustomAuditLog` which is called from many places in Fleet API handlers.